lib/tasks/test.js

```diff
@@ -67,7 +67,15 @@ module.exports = function(gulp, config) {
 		])
 		.then(function() {
 			if (config.browserTest) {
-				return module.exports.browserTest(gulp, config);
+				return module.exports.browserTest(gulp, config).then(function(){
```

Comment:
Ah, no worries, happy to help out with some tests if necessary when have some more free time on my hands